litmus-concepts.md

```diff
@@ -7,6 +7,8 @@ The main engineering goal of Litmus is to re-use as much existing content and ex
 Smaller re-usable components and leveraging functionality of other projects allows for easier adaptation and allows Litmus to ride along on the success of others, like bolt.
 It also means that it is easier to replace the parts that did not turn out to fulfill the needs of the users.
 
+![Litmus Overview]({% link resources/puppet-litmus.png %})
```

Comment:
Doesn't look like this is showing?

Reply:
try running `jekyll serve` to check this locally